MyPackages/my_funcs.py

```diff
@@ -27,4 +26,6 @@ def add_numbers(in0, in1):
         raise TypeError("First argument must be a integer or float")
     if not isinstance(in1, (int, float)):
         raise TypeError("Second argument must be a integer or float")
+
+    assert(in0 > -100)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/volker.hirthammer/pythonTest/pullRequest?prid=4663378)